Examples/Scripts/SetUpHCPPipeline.sh

```diff
@@ -93,7 +93,15 @@ then
 fi
 
 # Add the specified versions of some things to the front of $PATH, so we can stop using absolute paths everywhere
-export PATH="$CARET7DIR:$FSLDIR/bin:$PATH"
+if [[ -d "$FSLDIR/share/fsl/bin" ]] ; then
+    # For FSL 6.0.6 (release date: 22 Nov 2022) or later
+    export PATH="$FSLDIR/share/fsl/bin:$PATH"
+else
+    # For older versions of FSL
+    export PATH="$FSLDIR/bin:$PATH"
+fi
+export PATH="$CARET7DIR:$PATH"
+echo $PATH
```

Comment:
I don't know that we want it to echo the new `$PATH`, particularly without saying what it is that is being reported.

Reply:
Oops, I just included that for some testing locally. I never meant for it to be included. I've removed it in the latest commit.